qutebrowser/javascript/history.js

```diff
@@ -18,25 +18,24 @@
  */
 
 "use strict";
-
-window.loadHistory = (function() {
+window.loadHistory = (() => {
```

Comment:
Still not sure what to think about making every function a lambda - but I have no idea what's common here in the JS world.